### ISSUE TITLE
Ram Trucks _authApiKey and _authUrl update

### ISSUE THREAD

**Issue by Bart738** [wubbl0rz/FiatChamp#76]
I was able to pull this info during testing on my RAM, would we be able to update FiatClient.cs with the following info for RAM?

      _authApiKey = "fNQO6NjR1N6W0E5A6sTzR3YY4JGbuPv48Nj9aZci";
      _authUrl = "https://mfa.fcl-02.fcagcv.com";

![image](https://github.com/wubbl0rz/FiatChamp/assets/60965950/99630442-db45-42b7-972c-adba501e3639)

**Comment by wubbl0rz** [comment]
yes thx very good info :+1:  but im sorry i cannot update it right now. development is on hold until i find new motivation to work on it and im driving my fiat again.

**Comment by Combatchuck** [comment]
@Bart738 You have yours working? I forked and I see it get a 200 when connecting, but don't see it returning any vehicles.

[08:11:21 DBG] {
  "Userid": "SXMDUSEN_CO1906********92o",
  "Version": 0,
  "Vehicles": []
  
  Can you point me at the API docs? I can only find the location tracking one.

**Comment by Bart738** [comment]
Yes, making only the change above everything works.  Prior to making the change, everything worked except for the advance commands requiring the PIN.

**Comment by Combatchuck** [comment]
Thanks, I just re-purchased the add on, so it might take time to update to show the vehicle.

**Comment by Schocker360** [comment]
Were you able to get this to work @Combatchuck ?  I tried your repo in Home Assistant but it's still not pulling any of my vehicles.  I get the same as you posted above:

"Vehicles": []

Also, when I add your repo url, the url in the manage add-on repositories screen still says https://github.com/wubbl0rz/FiatChamp

**Comment by Combatchuck** [comment]
I realized that my 19 was not available via that api.

**Comment by Schocker360** [comment]
My 20 may not be available either.  I can start my truck through homebridge and the homekit integration but I can't lock it.

**Comment by Combatchuck** [comment]
connect.ramtrucks.com shows my truck on the top, but if I want to view it, it redirects me to www.mopar.com
so my truck isn't with ram trucks, its with mopar.
